server/data/visitSchedulerApiClient.ts

```diff
@@ -60,4 +60,23 @@ export default class VisitSchedulerApiClient {
       path: `/admin/session-templates/template/${reference}`,
     })
   }
+
+  async createSessionTemplate(sessionTemplate: CreateSessionTemplateDto): Promise<CreateSessionTemplateDto> {
```

Comment:
Although you do pass this a `CreateSessionTemplateDto`, it's return value is actually ` `SessionTemplateDto`.